.eslintrc

```diff
@@ -1,20 +1,22 @@
 {
-  "ecmaFeatures": {
-    "jsx": true,
-    "modules": true
-  },
+  "extends": "eslint-config-airbnb",
   "env": {
     "browser": true,
     "mocha": true,
     "node": true
   },
-  "parser": "babel-eslint",
```

Comment:
We already have it by (eslint-airbnb-config)[https://github.com/airbnb/javascript/blob/master/linters/.eslintrc#L2]